stdout.go

```diff
@@ -114,6 +114,16 @@ func (client *StdoutClient) PublishAuditLog(auditLogBuilder *AuditLogBuilder) er
 	return nil
 }
 
+func (client *StdoutClient) PublishMessage(topic, messageKey string, message interface{}, errorCallback PublishErrorCallbackFunc) error {
+	messageBuf, err := json.Marshal(&message)
+	if err != nil {
+		logrus.Errorf("unable to marshal message, error : %v", err)
+		return err
+	}
+	fmt.Println(string(messageBuf))
```

Comment:
is this intended to leave this? or should we remove this?

Reply:
@thoriqsatriya sorry, what do you mean?

Reply:
do we need to print the message in stdout?